.github/workflows/lint.yaml

```diff
@@ -28,7 +28,7 @@ jobs:
     - name: Lint
       run: |
         python --version
-        tox -e lint
+        tox -e lint,apidocs
```

Comment:
I think that instead of updating lint, we should fix the docs.yaml workflow to fail on pydoctor warnings

Reply:
we already have way too many workflows, I'd like to push more complexity into tox and out of all the .github files

Reply:
well...the docs.yaml just calls `tox -e apidocs` ... is just that for narrative docs and apidocs we have a separate workflow... so that for example if a flaky test fails we can re-run only the trial tests...without having to rerun lint and documentation generation

but yes.. I would say that mypy.yaml and docs.yaml and lint.yaml can be merged.

Reply:
I see that lint now takes 7 minutes... I think that one of the reason why docs was moved to a separate workflow, was to have the docs executed in a separate VM... so that we can get a fast feedback.

But maybe we can run tox on 2 CPUs ... or build the docs as a separate job inside the lint.yaml